<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ryzen TDP Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="Smoke_checkBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Enable Smokeless scale for TDP</string>
         </property>
         <property name="text">
          <string>Enable Smokeless scale</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Boost_checkBox">
         <property name="toolTip">
          <string>Override fast and slow boost limits</string>
         </property>
         <property name="text">
          <string>Boost limits override</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="4" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="slow_label">
         <property name="toolTip">
          <string>Slow Boost override</string>
         </property>
         <property name="text">
          <string>Slow</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="slow_tdp_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Slow Boost override</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slow_tdp_label">
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Watts</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="APU_tdp_label_2">
         <property name="toolTip">
          <string>ASUS ROG ALLY MCU Mode</string>
         </property>
         <property name="text">
          <string>ROG ALLY MCU Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MCU_Mode_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>ASUS ROG ALLY MCU Mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Refresh_pushButton">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="4" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="fast_label">
         <property name="toolTip">
          <string>Fast Boost override</string>
         </property>
         <property name="text">
          <string>Fast</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fast_tdp_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Fast Boost override</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fast_tdp_label">
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Watts</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" alignment="Qt::AlignBottom">
      <widget class="QLabel" name="APU_tdp_label">
       <property name="text">
        <string>APU TDP Setting</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QCheckBox" name="GPU_Clock_checkBox">
       <property name="toolTip">
        <string>Enable GPU Clock override</string>
       </property>
       <property name="text">
        <string>Enable GPU Clock Manual override</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="5" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="tdp_info_pushButton">
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>TDP info</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="GPU_Clock_Manual_label">
       <property name="text">
        <string>GPU Clock Manual Setting</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="About_pushButton">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="exit_pushButton">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="Governor_label">
       <property name="toolTip">
        <string>Scaling Governor</string>
       </property>
       <property name="text">
        <string>Scaling Governor</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="Governor_comboBox">
       <property name="toolTip">
        <string>Scaling Governor</string>
       </property>
       <item>
        <property name="text">
         <string>powersave</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>performance</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="EPP_label">
       <property name="toolTip">
        <string>Energy Performance Preference</string>
       </property>
       <property name="text">
        <string>Energy Pref</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" colspan="3">
      <widget class="QComboBox" name="EPP_comboBox">
       <property name="toolTip">
        <string>Energy Performance Preference</string>
       </property>
       <item>
        <property name="text">
         <string>balance_performance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>balance_power</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>performance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>power</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0" colspan="6">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="GPU_Clock_Slider">
         <property name="minimum">
          <number>800</number>
         </property>
         <property name="maximum">
          <number>2700</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="pageStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>2200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="GPU_Clock_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Manual GPU Clock value</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="GPU_Clock_label">
         <property name="text">
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GPU_Apply_pushButton">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Manual GPU override in effect until reboot...</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="6">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="tdp_Slider">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tdp_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>TDP value</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tdp_label">
         <property name="text">
          <string>Watts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tdp_Apply_pushButton">
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Apply TDP, thermal and EPP settings</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
